content/en/logs/processing/parsing.md

```diff
@@ -495,6 +496,63 @@ rule %{data::xml}
 * If the XML contains tags that have both an attribute and a sting value between the two tags, a `value` attribute is generated. For example: `<title lang="en">Harry Potter</title>` is converted to `{"title": {"lang": "en", "value": "Harry Potter" } }`
 * Repeated tags are automatically converted to arrays. For example: `<bookstore><book>Harry Potter</book><book>Everyday Italian</book></bookstore>` is converted to `{ "bookstore": { "book": [ "Harry Potter", "Everyday Italian" ] } }`
 
+### Parsing CSV
+
+Use the **csv** filter to more-easily map strings to attributes when separated by a given character (`,` by default).
+
+The csv filter is defined as `csv(headers[, separator[, quotingcharacter]])` where:
+
+* `headers`: defines the keys name separated by `,`. Keys names must start with alphabetical character and can contain any alphanumirical character in addition to `-`, `_`, `@` and `$`.
```

Comment:
```suggestion
* `headers`: Defines the keys name separated by `,`. Keys names must start with alphabetical character and can contain any alphanumerical character in addition to `-`, `_`, `@` and `$`.
```

Reply:
Shouldn't it be `alphanumeric` instead of `alphanumerical`?